### PR TITLE
Remove unit_count parameter doc for parameter that no longer exists

### PR DESCRIPTION
**Pull request by dashford**
Was still showing up in the documentation even though the `unit_count` parameter has been removed.

### CODE REVIEW

**Comment by untergeek** [comment]
Please change this to be committed against the `5.x` branch, rather than `master`.  Huge changes have already been made to my local code branch for `master`, and this will actually conflict there.

Thanks!

**Comment by dashford** [comment]
Hey @untergeek, I've made those changes now - let me know if there's anything else needed here.

File: setup.py [hunk 1]
```diff
@@ -42,7 +42,6 @@ def get_install_requires():
 
 
     base = 'Console'
-    msvcrt = ''
```
Comment (untergeek) [inline setup.py]: Why is this causing issues for you?

Comment (untergeek) [reply setup.py]: Nevermind.  That was one of the other commits from changing base.

**Pull request merged**